tests/client/test_dataset.py

```diff
@@ -600,6 +615,45 @@ def test_prepare_for_training_with_spacy(self):
         assert isinstance(train, spacy.tokens.DocBin)
         assert len(train) == 100
 
+        train, test = rb_dataset.prepare_for_training(
```

Comment:
It would be nice to add also some tests to increase the overall coverage (train_size + test_size != 1, no train/test size with spark_nlp)

Reply:
Thansk, I added this too.